source/2_platform/rex_windows/src/entrypoint.cpp

```diff
@@ -22,24 +32,37 @@ int rex_entry(HINSTANCE hInstance, HINSTANCE hPrevInstance, LPTSTR lpCmdLine, in
   creation_params.prev_instance = hPrevInstance;
   creation_params.show_cmd      = nShowCmd;
 
-  rex::ApplicationCreationParams app_params = rex::app_entry(rsl::move(creation_params));
-
   s32 result = 0;
-  if(app_params.create_window)
-  {
-    // this doesn't initialize anything but simply prepares the application for initialization
-    rex::win32::GuiApplication application(rsl::move(app_params));
 
-    // this initializes, runs the loop and performs the shutdown
-    result = application.run();
+  // We're wrapping executing in __try - __except block so that we can capture crashes
+  // This allows us to perform particular logic when we encounter a crash.
+  // Especially for when running tests, this can be very useful
+  __try
```

Comment:
I think it wasn't recommended to use this, but I think there isn't really an alternative.